alarm clock/alarm clock/Form1.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Timers;
+using System.Windows.Forms;
+
+namespace alarm_clock
+{
+    public partial class Form1 : Form
+    {
+       
+        public Form1()
+        {
+            InitializeComponent();
+        }
+        int hour, minute, second;
+        string alarmHour, alarmMinute;
+
+        private void Timer1_Tick(object sender, EventArgs e)
+        {
+            second = DateTime.Now.Second;
+            minute = DateTime.Now.Minute;
+            hour = DateTime.Now.Hour;
+            label2.Text = hour.ToString();
+            label3.Text = minute.ToString();
+            label4.Text = second.ToString();
+            ring_alarm();
+           
+        }
+
+        void ring_alarm()
+        {
+            if(alarmHour==hour.ToString() && alarmMinute == minute.ToString() && second.ToString == "0")
```

Comment:
Ошибка. Не хватает скобок.
```suggestion
            if(alarmHour==hour.ToString() && alarmMinute == minute.ToString() && second.ToString() == "0")
```